parser_test.go

```diff
@@ -2723,6 +2723,23 @@ func (s *testParserSuite) TestOptimizerHints(c *C) {
 	c.Assert(hints[1].Indexes, HasLen, 1)
 	c.Assert(hints[1].Indexes[0].L, Equals, "c1")
 
+	// Test READ_CONSISTENT_STORAGE
+	stmt, _, err = parser.Parse("select /*+ READ_CONSISTENT_STORAGE(tiflash[t1, t2], tikv[t3]) */ c1, c2 from t1, t2, t1 t3 where t1.c1 = t2.c1 and t2.c1 = t3.c1", "", "")
```

Comment:
Why choose this syntax? Why not `READ_CONSISTENT_STORAGE(tiflash, t1, t2), READ_CONSISTENT_STORAGE(tikv,t3)` to be consistent with others？

Reply:
Because we can regard a `tiflash[t1, t2]` as a whole. If the "query block" also exists in this hint, the hint will like `READ_CONSISTENT_STORAGE(@sel1 tiflash t1, t2)`. That's incomprehensible.

Reply:
Also seems incomprehensible  in this case: `READ_CONSISTENT_STORAGE(tiflash[t1, t2], tikv[t3])` ----> `READ_CONSISTENT_STORAGE(tiflash t1, t2 tikv t3)`

Reply:
- I don't have strong opinion on this specific syntax, but if you decided to group similiar things, it is better to change all others that has similiar syntax, like `index`.
- I think it is quite clear what the comment want to say in `READ_CONSISTENT_STORAGE(@sel1 tiflash t1, t2)`.

Reply:
No, just seperate them into two.

Reply:
Hmmm...So may I keep this format? The product manager says that we can only focus on functions. The form is under consideration now.